jwql/tests/test_api_views.py

```diff
@@ -42,23 +42,32 @@
     urls.append('api/{}/proposals/'.format(instrument))  # instrument_proposals
 
 # Proposal-specific URLs
-proposals = ['01018',  # FGS
-             '01022',  # MIRI
-             '01068',  # NIRCam
-             '01059',  # NIRISS
-             '1059',  # NIRISS
-             '01106']  # NIRSpec
+
+# TODO: currently missing in central storage
+# proposals = ['01018',  # FGS
+#              '01022',  # MIRI
+#              '01068',  # NIRCam
+#              '01059',  # NIRISS
+#              '1059',  # NIRISS
+#              '01106']  # NIRSpec
+proposals = ['2640', '02733', '1541', '02589']
```

Comment:
I think this is a good set to test with. JWQL is designed to prepend a 0 to proposal numbers that have only 4 digits. So it should translate 2640 to 02640, and 02733 should stay as 02733. It's good to confirm that's working. In terms of real programs (commissioning and cycle 1) that have data, we're not much beyond 2733. We used to have fake data with program numbers >10000, but we probably shouldn't touch that data any more, especially because none of that is in MAST.